addons/website/tests/test_ui.py

```diff
@@ -5,6 +5,41 @@
 import odoo.tests
 
 
+@odoo.tests.tagged('-at_install', 'post_install')
+class TestUiCustomizeTheme(odoo.tests.HttpCase):
+    def test_01_attachment_website_unlink(self):
```

Comment:
attachment is now specific usecase,=>  + add test on other model than attchmnet

Reply:
Test adapted, check that a PDF from a SO is not removed but only website_id empty, and a scss custo is removed.